woocommerce/woocommerce-ios
# [Shipping labels] updates for star/unstar action

#14598


## Description


- using `WooShippingAction.createPackage` to star predefined package and processing response
- missing ability to unstar a package, this will be done with the delete action from https://github.com/woocommerce/woocommerce-ios/pull/14591 when it is ready

## Testing information


- Install and set up the Woo Shipping extension on your store.
- Build and run the app with the revampedShippingLabelCreation feature flag enabled.
- Create an order with the processing status and at least one physical product.
- In the order details, select "Create Shipping Label."
- Store options will start fetching from backend in the background.
- Tap "Select a Package" button
- Switch to Carrier tab
- Tap on star icon (not filled) for one of the packages
- Switch to Saved tab and check that the starred/saved packages is in that list

## Screenshots


https://github.com/user-attachments/assets/88fc8d81-f53e-4467-a6ba-344298fb7a19

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
